roles/ceph-osd/tasks/scenarios/dmcrypt-dedicated-journal.yml

```diff
@@ -23,6 +23,24 @@
     - item.0.get("rc", 0) != 0
     - item.1.get("rc", 0) != 0
     - not osd_auto_discovery
+    - osd_objectstore == 'filestore'
```

Comment:
On Luminous, `ceph-disk prepare` default is Bluestore, so we need to change the default of `osd_objectstore` I suppose.